services/ui-src/src/libs/formLib.tsx

```diff
@@ -14,7 +15,7 @@ export type OneMACFormConfig = {
   addlIntroJSX?: JSX.Element;
   detailsHeader?: string;
   landingPage: string;
-  confirmSubmit: boolean;
+  confirmSubmit?: ConfirmSubmitType;
```

Comment:
I like how you turned the boolean into the details about the confirmation message!